azerothcore/mod-autobalance
# Fix for broken chat messages #187

## Changes Proposed:
- Fixes the issue that variables are not properly replaced. Instead placeholders %s,%u and %i were shown in chat.

## Issues Addressed:

- Closes issue #187

## SOURCE:


## Tests Performed:

- Builds without error on Windows
- Tested on locally compiled instance


## How to Test the Changes:


1. Install module and log into game
2. Enter any instance
3. Enter command .ab mapstat and check if message is displayed properly
4. Especially check if you notice any placeholders not being replace %s, %u, %i
